Remove unused `stroke` attributes from SVG logos

guihkx opened the pull request:
This prevents warnings in Qt apps using the QtSvg library when displaying the affected SVG logos. The warning:

```
qt.svg: <input>:LINE:COLUMN: Could not resolve property: #linearGradient10962
```

On Linux, for example, you'll see that warning by opening any of the affected SVGs with [Gwenview](https://apps.kde.org/gwenview/) from the command line. You can also occasionally see it in the system journal when KDE Plasma loads the Alacritty logo for the first time to display it on the taskbar.

That `#linearGradient10962` property does not exist in any of the SVGs, so this removes all references to it in `stroke` attributes.

This also removes other unused `stroke-*` attributes that do not affect the SVGs visually.

chrisduerr reviewed (changes requested):
I'm a bit surprised about this because the simple Qt SVG renderer generally struggles to render Alacritty's SVGs, since it doesn't support clip paths.

Are you familiar with that issue, or is this just something completely separate you noticed?

guihkx commented:
>I'm a bit surprised about this because the simple Qt SVG renderer generally struggles to render Alacritty's SVGs, since it doesn't support clip paths.

I wasn't aware of that at all... To be completely honest, all SVGs look identical to me when comparing them side-by-side in Gwenview and Inkscape (which I'm assuming does have support for clip paths).

I only noticed that warning (actually a bunch of it) while I was searching for an unrelated issue in my system logs.

I don't know anything about SVGs either, but I `grep`'ed for other `linearGradient*` references  in the SVG, and with the exception of `#linearGradient10962`, they all were somewhat defined in the SVG like this:

```xml
<linearGradient id="linearGradient*" osb:paint="solid"></linearGradient>
```

So I thought: `#linearGradient10962` must've been a leftover of an experiment, or something like that. 

Next, I opened each SVG with Inkscape, removed the offending `#linearGradient10962` from the `style` attribute in the appropriate `<path>` tag (and also removed other `stroke` attributes that I thought to be leftovers as well).

And because I saw no visual changes in Inkscape after these changes, I thought this was correct/harmless, and so I sent this PR.

Is this a mistake?

chrisduerr commented:
Yeah it's super obvious because a big chunk of Alacritty's icon will just be a white triangle. There's a bug in the QSvgGenerator that has been declared out of scope: https://bugreports.qt.io/browse/QTBUG-1865.

That's why I was curious if that is actually showing up properly, because there's no reason to make this modification if it will just fail for a different reason with it applied. Can you share a screenshot of what it looks like with your patch inside Gwenview?

chrisduerr commented:
> Unless this #linearGradient10962 thing is actually being used somewhere and QtSvg just doesn't know what to do with it. In which case, feel free to close this.

It's probably unused and doesn't hurt anything to remove it. But considering that it doesn't actually *help* anyone, it's probably best to just not change anything. I don't think we need to worry about size of these files.
